bot/cogs/define_cog.py

```diff
@@ -0,0 +1,150 @@
+
+                except:
+                    pass
+                
+                #Type of the given word (Noun, Verb, Adjective, etc.)
+                wordType = ""
+                try:
+                    wordType = wordData['fl']
+                except:
+                    pass
+                
+                #Definitions of the given word
+                definitions = []
+                try:
+                    defData = wordData['shortdef']
+                    for defin in range(len(defData)):
+                        definitions.append(defData[defin])
+                except:
+                    definitions.append("Definition Missing")
+                
+                #Turn data into one long string (represents a page)
+                template = "Tenses: "
+                for s in range(len(wordStems)):
+                    template += wordStems[s]
+                    if (s != len(wordStems) - 1):
+                        template += ", "
+                template += "\n"
+
+                template += "Syllables: " + syllableData + "\n"
+
+                template += "Pronunciation: "
+                for s in range(len(pronunc)):
+                    template += pronunc[s]
+                    if (s != len(pronunc) - 1):
+                        template += ", "
+                template += "\n"
+
+                template += "Word Type: " + wordType + "\n"
+
+                template += "\n"
+                for s in range(len(definitions)):
+                    page = template + "Definition: " + definitions[s]
+                    page = page.replace("*"," | ")
+                    pages.append(page)
+
+        #If the word cannot be found, the JSON returns a list of other possible suggestions. 
+        elif (isinstance(jsonData[0], str)):
+            template = "Word not found, see also: "
+            for s in range(len(jsonData)):
+                template += jsonData[s]
+                if (s != len(jsonData) - 1):
+                    template += ", "
+    
+            pages = [template]
+
+        return pages
+    
+    @commands.command()
+    async def define(self, ctx, word):
+        """
+        Given a word, find its definition and any other relevant information
+        
+        USE: !define <word>
+        EXAMPLE: !define schadenfreude
+
+        For phrases, use underscores
+        EXAMPLE: !define computer_science
+        """
+
+        actualWord = word.replace("_"," ")
+        word = word.replace("_","%20")
+        url = "https://www.dictionaryapi.com/api/v3/references/collegiate/json/" + word.lower() + "?key=" + self.api_key
+        response = requests.get(url)
+        
+        if (response.status_code == 200):
+            self.wordPages = self.getPageData(response, word)
+
+        else:
+            ErrMsg = "Oh No! There appears to be an issue! Yell at one of the developers with the following code.\nError Code: " + str(response.status_code)
```

Comment:
the error message will need to be an embed with the Error color defined in bot.consts